README.md

```
@@ -1,33 +1,34 @@
 # ⚡️piking Neural Network - A Rust Implementation 🦀 
 
 We consider (continuous-time) spiking neural networks and their use as robust memorizers of arbitrary spike trains. 
-For details, we refer to [Aguettaz and Loeliger, 2024](https://arxiv.org/abs/2408.01166). 
-
-## Why Rust? 
+We refer to [Aguettaz and Loeliger, 2024](https://arxiv.org/abs/2408.01166) for details. 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)